pkg/segment/query/segquery.go

```diff
@@ -757,15 +747,10 @@ func getAllRotatedSegmentsInAggs(queryInfo *QueryInformation, aggs *structs.Quer
 	qsrs, err := applyQsrsFilterHook(qsrs, true)
 	if err != nil {
 		log.Errorf("getAllRotatedSegmentsInAggs: qid=%d, failed to apply hook: %v", queryInfo.qid, err)
-		return nil, 0, 0, err
+		return nil, 0, err
 	}
 
-	numDistributedRequests, err := queryInfo.dqs.DistributeRotatedRequests(queryInfo)
-	if err != nil {
-		log.Errorf("getAllRotatedSegmentsInAggs: qid=%d, Error distributing rotated requests %+v", queryInfo.qid, err)
-		return nil, 0, 0, err
-	}
-	return qsrs, totalQsr - numDistributedRequests, numDistributedRequests, nil
```

Comment:
I am not sure why we are subtracting the numDistributedRequests from totalQsr over here, but not doing the same thing for unrotated or for rotated and unrotated in the getAllSegmentsInQuery. I ran the tests and seemed to be working fine.

Reply:
I'm not sure either. I think you're right to remove it